rlp/src/lib.rs

```diff
@@ -33,9 +33,13 @@
 //! * You don't want to decode whole rlp at once.
 
 extern crate byteorder;
+#[cfg(feature = "ethereum")]
 extern crate ethereum_types as bigint;
```

Comment:
bigint? lol. It's probably done by me when refactoring and I forgot to remove the alias